media-sound/mpd/mpd-0.20.6.recipe

```diff
@@ -7,7 +7,7 @@ COPYRIGHT="2003-2017 The Music Player Daemon Project"
 LICENSE="GNU GPL v2"
 REVISION="1"
 SOURCE_URI="http://www.musicpd.org/download/mpd/0.20/mpd-$portVersion.tar.xz"
-CHECKSUM_SHA256="525573be27443ae26868b21a61e88c03909e72f1212ddaa725a0389a28d88987"
+CHECKSUM_SHA256="f4055e1189e5cc00e83706b2e0b5ead924ced0303e073d7802ee9f9a8eba4b47"
 PATCHES="mpd-0.20.5.patchset"
 
```

Comment:
It seems upstream makes releases on http://git.musicpd.org/cgit/master/mpd.git/log/?h=v0.20.x
This branch doesn't have the patch. Only 0.21.0 will get the patch.